### ISSUE TITLE
Support <dirset> in chmod, chown, delete, echo, copy, foreach and move tasks (Trac #1026)

### ISSUE THREAD

**Issue by phing-issues-importer** [phingofficial/phing#559]
In addition to `<fileset>`, phing has a `<dirset>` task that's currently only used as subtag for `<path>` tags.

It'd be nice if one could use it in other tasks, too:
- chmod
- chown
- delete
- echo
- copy
- foreach
- move

Migrated from https://www.phing.info/trac/ticket/1026

``` json
{
    "status": "new", 
    "changetime": "2014-09-18T12:47:57", 
    "description": "In addition to `<fileset>`, phing has a `<dirset>` task that's currently only used as subtag for `<path>` tags.\n\nIt'd be nice if one could use it in other tasks, too:\n\n* chmod\n* chown\n* delete\n* echo\n* copy\n* foreach\n* move\n", 
    "reporter": "cweiske", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1411044477789493", 
    "component": "phing-tasks-system", 
    "summary": "Support <dirset> in chmod, chown, delete, echo, copy, foreach and move tasks", 
    "priority": "low", 
    "keywords": "", 
    "version": "2.5.0", 
    "time": "2013-07-25T08:29:00", 
    "milestone": "Backlog", 
    "owner": "mrook", 
    "type": "enhancement"
}
```

**Comment by phing-issues-importer** [comment]
Trac update at 20130818T09:32:58: 
* mrook commented:

> Good idea!


* mrook changed milestone from "Backlog" to "2.7.0"
* mrook changed priority from "tbd" to "minor"

**Comment by phing-issues-importer** [comment]
Trac update at 20131212T10:36:23: mrook changed milestone from "2.7.0" to "2.8.0"

**Comment by phing-issues-importer** [comment]
Trac update at 20140918T12:47:57: 
* mrook changed milestone from "2.9.0" to "Backlog"
* mrook changed priority from "minor" to "low"

**Comment by mrook** [comment]
Implemented by #674, #675, #678, #679 and #680.
